features/step_definitions/fixture_steps.rb

```diff
@@ -46,6 +46,11 @@
   Host.first.site.delete
 end
 
+Given(/^there is a site called (.*) belonging to an organisation ([^ ]*)$/) do |site_abbr, org_abbr|
+  org = create(:organisation, title: org_abbr, whitehall_slug: org_abbr)
+  create(:site, organisation: org, abbr: site_abbr)
+end
```

Comment:
The existing [`Given(/^a site (.*) exists$/)`](https://github.com/alphagov/transition/blob/a305b6d436ec30ab9146380f08fd9e179cfbcce6/features/step_definitions/fixture_steps.rb#L98-L100) step does pretty much this already, so you should be able to use that instead of introducing a new step. The `site` factory has `association :organisation` so will create an org too, which is enough to make the site dashboard page work.